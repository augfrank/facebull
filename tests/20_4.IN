M0   C0   C1  2612
M1   C0   C2  2794
M2   C0   C3  2488
M3   C0   C5  2953
M4   C0   C6  2570
M5   C0   C7  2181
M6   C0   C8  2043
M7   C0   C12  2997
M8   C0   C15  2576
M9   C0   C16  2504
M10   C0   C19  2996
M11   C1   C2  2672
M12   C1   C3  2507
M13   C1   C4  2697
M14   C1   C8  2335
M15   C1   C11  2871
M16   C1   C18  2074
M17   C2   C0  2352
M18   C2   C3  2206
M19   C2   C4  2955
M20   C2   C5  2700
M21   C2   C9  2391
M22   C2   C11  2017
M23   C2   C12  2794
M24   C2   C14  2347
M25   C2   C16  2863
M26   C2   C19  2705
M27   C3   C1  2224
M28   C3   C2  2387
M29   C3   C4  2264
M30   C3   C5  2491
M31   C3   C10  2759
M32   C3   C12  2711
M33   C3   C14  2616
M34   C4   C1  2835
M35   C4   C2  2412
M36   C4   C3  2565
M37   C4   C6  2469
M38   C4   C7  2240
M39   C4   C10  2600
M40   C4   C11  2344
M41   C4   C15  2069
M42   C4   C16  2341
M43   C4   C17  2496
M44   C5   C0  2410
M45   C5   C1  2002
M46   C5   C2  2590
M47   C5   C6  2599
M48   C5   C7  2797
M49   C5   C8  2868
M50   C5   C9  2136
M51   C5   C10  2935
M52   C5   C12  2415
M53   C5   C13  2452
M54   C5   C14  2678
M55   C5   C15  2965
M56   C5   C19  2240
M57   C6   C2  2643
M58   C6   C4  2195
M59   C6   C10  2005
M60   C6   C11  2092
M61   C6   C12  2024
M62   C6   C13  2897
M63   C6   C15  2039
M64   C6   C17  2812
M65   C6   C18  2322
M66   C7   C0  2172
M67   C7   C1  2399
M68   C7   C5  2128
M69   C7   C8  2913
M70   C7   C9  2646
M71   C7   C10  2666
M72   C7   C12  2284
M73   C7   C15  2351
M74   C8   C0  2662
M75   C8   C1  2727
M76   C8   C4  2814
M77   C8   C6  2824
M78   C8   C7  2426
M79   C8   C10  2648
M80   C8   C12  2198
M81   C8   C18  2767
M82   C9   C4  2987
M83   C9   C7  2045
M84   C9   C11  2934
M85   C9   C13  2122
M86   C9   C15  2062
M87   C9   C16  2596
M88   C9   C18  2768
M89   C9   C19  2122
M90   C10   C2  2256
M91   C10   C3  2863
M92   C10   C5  2920
M93   C10   C11  2207
M94   C10   C13  2458
M95   C10   C14  2800
M96   C10   C18  2635
M97   C10   C19  2059
M98   C11   C1  2135
M99   C11   C2  2222
M100   C11   C4  2336
M101   C11   C6  2052
M102   C11   C16  2447
M103   C11   C17  2258
M104   C12   C0  2222
M105   C12   C1  2998
M106   C12   C6  2814
M107   C12   C8  2313
M108   C12   C11  2617
M109   C12   C13  2055
M110   C12   C15  2685
M111   C12   C16  2005
M112   C13   C0  2219
M113   C13   C1  2651
M114   C13   C3  2210
M115   C13   C4  2356
M116   C13   C7  2177
M117   C13   C8  2978
M118   C13   C9  2713
M119   C13   C10  2387
M120   C13   C14  2507
M121   C13   C19  2362
M122   C14   C1  2218
M123   C14   C4  2624
M124   C14   C7  2202
M125   C14   C9  2635
M126   C14   C10  2780
M127   C14   C12  2813
M128   C14   C13  2091
M129   C15   C1  2308
M130   C15   C2  2731
M131   C15   C6  2256
M132   C15   C7  2725
M133   C15   C8  2082
M134   C15   C9  2176
M135   C15   C12  2568
M136   C15   C14  2843
M137   C15   C17  2550
M138   C15   C19  2612
M139   C16   C0  2306
M140   C16   C1  2978
M141   C16   C2  2856
M142   C16   C3  2514
M143   C16   C6  2234
M144   C16   C7  2764
M145   C16   C8  2076
M146   C16   C9  2927
M147   C16   C10  2801
M148   C16   C11  2594
M149   C16   C13  2987
M150   C16   C14  2980
M151   C16   C15  2094
M152   C16   C18  2526
M153   C17   C5  2014
M154   C17   C7  2254
M155   C17   C8  2665
M156   C17   C9  2555
M157   C17   C13  2840
M158   C17   C15  2102
M159   C17   C19  2027
M160   C18   C1  2915
M161   C18   C2  2239
M162   C18   C3  2177
M163   C18   C4  2273
M164   C18   C8  2688
M165   C18   C11  2541
M166   C18   C15  2123
M167   C18   C19  2099
M168   C19   C3  2947
M169   C19   C5  2199
M170   C19   C8  2536
M171   C19   C9  2659
M172   C19   C11  2074
M173   C19   C12  2936
M174   C19   C13  2782
M175   C19   C14  2745
M176   C19   C15  2279
M177   C19   C16  2294
M178   C19   C17  2030
M179   C19   C18  2081
