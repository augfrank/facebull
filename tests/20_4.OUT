42579
6 16 24 29 41 45 59 66 80 83 97 103 111 114 128 134 143 153 161 172
